### PR TITLE
Abravosuse version 2.2.0

### PR DESCRIPTION
**Pull request by abravosuse**
Updating documentation to version 2.2.0.

### CODE REVIEW

File: trento/xml/article_sap_trento.xml [hunk 1]
```diff
@@ -962,12 +962,30 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
...
           <formalpara>
-            <title><guimenu>SAP instances</guimenu>:</title>
-            <para>it summarizes the status of the ABAP instances as returned by <command>sapcontrol</command>.</para>
+            <title><guimenu>Application instances</guimenu>:</title>
+            <para>it summarizes the status of the application instances as returned by <command>sapcontrol</command>.</para>
+          </formalpara>
+        </listitem>
+      </itemizedlist>
+      <para>
+        Complementing the Operating System layer, you also have information about the health status of the HA components, whenever they exit:
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:971]: Is "Operating System Layer" a specific SAP term? If not, it should be lower case:

```suggestion
        Complementing the operating system layer, you also have information about the health status of the HA components, whenever they exit:
```

File: trento/xml/article_sap_trento.xml [hunk 2]
```diff
@@ -1021,19 +1039,25 @@ The health boxes in the dashboard are clickable. By clicking on one particular b
          </listitem>
          <listitem>
           <formalpara>
-            <title><guimenu>Cluster health icon</guimenu>:</title>
-            <para>Link to the corresponding &pace; Cluster Details view.</para>
+            <title><guimenu>Database cluster health icon</guimenu>:</title>
+            <para>Link to the corresponding HANA Cluster Details view.</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1043]: Use entity:

```suggestion
            <para>Link to the corresponding &hana; Cluster Details view.</para>
```

File: trento/xml/article_sap_trento.xml [hunk 3]
```diff
@@ -1021,19 +1039,25 @@ The health boxes in the dashboard are clickable. By clicking on one particular b
...
+          <formalpara>
+            <title><guimenu>Application cluster health icon</guimenu>:</title>
+            <para>Link to the corresponding ASCS/ERS Cluster Details view.</para>
+          </formalpara>
          </listitem>
          <listitem>
           <formalpara>
-            <title><guimenu>SAP Instances health icon</guimenu>:</title>
+            <title><guimenu>Application Instances health icon</guimenu>:</title>
             <para>Link to the corresponding SAP System Details view.</para>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1061]: Use entity:

```suggestion
            <para>Link to the corresponding &sap; System Details view.</para>
```

File: trento/xml/article_sap_trento.xml [hunk 4]
```diff
@@ -1097,8 +1121,38 @@ The health boxes in the dashboard are clickable. By clicking on one particular b
...
                 <para><guimenu>Hosts details</guimenu> section: it shows the
                   status of both the &t.agent; and the Node Exporter and
-                  provides the host name, the cluster name (when applicable) and
-                  the &t.agent; version. </para>
+                  provides the host name, the cluster name (when applicable),
+                  the &t.agent; version and the host IP addresses. </para>
+              </listitem>
+              <listitem>
+                <para>saptune summary section: saptune is a solution that comes with SLES for SAP
+                 and allows SAP admins to ensure that their SAP host are properly configured to run the corresponding SAP workloads.
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1129]: Use entity:

```suggestion
                <para>saptune summary section: saptune is a solution that comes with &sles5sap;
                 and allows &sap; administrators to ensure that their &sap; host are properly configured to run the corresponding &sap; workloads.
```

File: trento/xml/article_sap_trento.xml [hunk 5]
```diff
@@ -1097,8 +1121,38 @@ The health boxes in the dashboard are clickable. By clicking on one particular b
...
+                 The integration of saptune in the Trento console gives the SAP admin visibility over the tool even when they are not
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1130]: Use entity (and admin -> administrator):

```suggestion
                 The integration of saptune in the Trento console gives the &sap; administrator visibility over the tool even when they are not
```

File: trento/xml/article_sap_trento.xml [hunk 6]
```diff
@@ -1097,8 +1121,38 @@ The health boxes in the dashboard are clickable. By clicking on one particular b
...
+        <mediaobject>
+          <imageobject>
+            <imagedata width="80%" fileref="saptune-summary-section.png" />
+          </imageobject>
+        </mediaobject>
+      </figure>
+                 <para>If an SAP workload is running on the host but no saptune or a version lower than 3.1.0 is installed, a warning
+                 is added to the aggregated health status of the host. When saptune version 3.1.0 or higher, is installed, a details view
+                 is also available showing detailed information about the saptune status:</para>
+                       <figure xml:id="fig-saptune-details-view">
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1144]: Just a friendly warning. I usually try to avoid adding version information as much as possible. They get outdated very quickly and becomes a maintenance nightmare. Imagine you have many of these texts. You would then have to check each one to see if it is still up to date.

In this case, it may be justified. Probably it's not possible, but could you rephrase this paragraph? If not, leave it as is.

File: trento/xml/article_sap_trento.xml [hunk 7]
```diff
@@ -1428,6 +1474,32 @@ The health boxes in the dashboard are clickable. By clicking on one particular b
        For example, if the user cleans up the host where the primary application server of an SAP System is registered,
        the entire SAP System will be removed from the console.
       </para>
+      <para>Similalirly, when a registered application or HANA instance is no longer discovered, an option to clean it up
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1477]: Typo and entity:

```suggestion
      <para>Similarly, when a registered application or &hana; instance is no longer discovered, an option to clean it up
```

File: trento/xml/article_sap_trento.xml [hunk 8]
```diff
@@ -1428,6 +1474,32 @@ The health boxes in the dashboard are clickable. By clicking on one particular b
...
+       will show up both in the corresponding overview and the corresponding details view.
+      </para>
+           <figure>
+              <title>Clean up button SAP systems overview</title>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1481]:
```suggestion
              <title>Clean up button &sap; systems overview</title>
```

File: trento/xml/article_sap_trento.xml [hunk 9]
```diff
@@ -1428,6 +1474,32 @@ The health boxes in the dashboard are clickable. By clicking on one particular b
...
+              <mediaobject>
+                <imageobject>
+                  <imagedata fileref="trento-cleanup-sap-instance-overview.png"
+                    width="70%" />
+                </imageobject>
+              </mediaobject>
+            </figure>
+                 <figure>
+              <title>Clean up button in SAP system details view</title>
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1490]:
```suggestion
              <title>Clean up button in &sap; system details view</title>
```

File: trento/xml/article_sap_trento.xml [hunk 10]
```diff
@@ -1428,6 +1474,32 @@ The health boxes in the dashboard are clickable. By clicking on one particular b
...
+              <mediaobject>
+                <imageobject>
+                  <imagedata fileref="trento-cleanup-sap-instance-details-view.png"
+                    width="70%" />
+                </imageobject>
+              </mediaobject>
+            </figure>
+           <para>When the user clicks on the cleanup button, a box will open asking for confirmation.
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1498]: "Clicks on" should be avoided, therefor:

```suggestion
           <para>When the user clicks the cleanup button, a box will open asking for confirmation.
```

File: trento/xml/article_sap_trento.xml [hunk 11]
```diff
@@ -1428,6 +1474,32 @@ The health boxes in the dashboard are clickable. By clicking on one particular b
...
+       If the user grants it, the instance will be removed from the console along with any possible dependencies
+       For example, if the user cleans up the ASCS instance of an SAP System,
+       the entire SAP System will be removed from the console.
```
Comment (tomschr) [inline trento/xml/article_sap_trento.xml:1501]: Entity and spelling:

```suggestion
       For example, if the user cleans up the ASCS instance of an &sap; system,
       the entire &sap; system will be removed from the console.
```

**Review by tomschr — CHANGES_REQUESTED** [review]
Thanks @abravosuse, great job! :+1:
I have some minor suggestions for you. :slightly_smiling_face: Hope they make sense.

There is also a validation error in regards to a table. I can have a look once you've finished with my suggestions.

File: trento/xml/article_sap_trento.xml (hunk 6 above)
Comment (abravosuse) [reply trento/xml/article_sap_trento.xml:1144]: saptune version 3.1.0 is a reference here. We cannot remove it.

File: trento/xml/article_sap_trento.xml [hunk 12]
```diff
@@ -1097,8 +1121,38 @@ The health boxes in the dashboard are clickable. By clicking on one particular b
               <listitem>
                 <para><guimenu>Hosts details</guimenu> section: it shows the
                   status of both the &t.agent; and the Node Exporter and
-                  provides the host name, the cluster name (when applicable) and
-                  the &t.agent; version. </para>
+                  provides the host name, the cluster name (when applicable),
+                  the &t.agent; version and the host IP addresses. </para>
+              </listitem>
+              <listitem>
+                <para>saptune summary section: saptune is a solution that comes with &sles5sap;
```
Comment (abravosuse) [inline trento/xml/article_sap_trento.xml:1128]:
```suggestion
                <para>saptune summary section: saptune is a solution that comes with &sles4sap;
```

**Comment by tomschr** [comment]
@abravosuse Here is the PDF after the fix of the table:

[SLES-SAP-trento_en.pdf](https://github.com/SUSE/doc-unversioned/files/13501015/SLES-SAP-trento_en.pdf)

**Review by tomschr — APPROVED** [review]
According to Alberto, Trento 2.2.0 is out. Ready to be published.
